### PR TITLE
Remove protocol fee

### PR DESCRIPTION
**Pull request by andreivladbrg**
As discussed in https://github.com/sablier-labs/company-discussions/discussions/2, this PR removes the protocol fees.

With this change, it was possible to remove:

- `SablierV2Comptroller` contract
- `SablierV2Base` contract

### CODE REVIEW

**Review by smol-ninja — APPROVED** [review]
Looks good. I have rebased from staging and added a commit: https://github.com/sablier-labs/v2-core/compare/2eff9279ab7429267448bbf8d4807b32b3addb16..0e218d7cf44e4f0f7daedd8c3cb4fe95b9395757 that removes unused imports and fixes some NatSpec. Kindly review before merging.

File: src/interfaces/ISablierV2LockupDynamic.sol [hunk 1]
```diff
@@ -18,8 +18,8 @@ interface ISablierV2LockupDynamic is ISablierV2Lockup {
     /// @param funder The address which has funded the stream.
     /// @param sender The address from which to stream the assets, who will have the ability to cancel the stream.
     /// @param recipient The address toward which to stream the assets.
-    /// @param amounts Struct containing (i) the deposit amount, (ii) the protocol fee amount, and (iii) the
-    /// broker fee amount, all denoted in units of the asset's decimals.
+    /// @param amounts Struct containing (i) the deposit amount, and (ii) the broker fee amount, all denoted
```
Comment (PaulRBerg) [inline src/interfaces/ISablierV2LockupDynamic.sol:21]: Let's say "both denoted".

File: src/interfaces/ISablierV2Lockup.sol [hunk 2]
```diff
@@ -131,6 +132,11 @@ interface ISablierV2Lockup is
     /// @param streamId The stream id for the query.
     function isWarm(uint256 streamId) external view returns (bool result);
 
+    /// @notice Retrieves the maximum broker fee that can be charged by the broker, denoted as a fixed-point
+    /// number where 1e18 is 100%.
+    /// @dev This value is hard coded as a constant.
+    function MAX_BROKER_FEE() external view returns (UD60x18);
```
Comment (PaulRBerg) [inline src/interfaces/ISablierV2Lockup.sol:138]: Shouldn't this have remained in `SablierV2Base`?

**Review by PaulRBerg — CHANGES_REQUESTED** [review]
good job

Haven't looked at the tests but `src` looks good

File: src/interfaces/ISablierV2Lockup.sol (hunk 2 above)
Comment (andreivladbrg) [reply src/interfaces/ISablierV2Lockup.sol:138]: `SablierV2Base` no longer exists - keeping the contract only for one constant made no sense.

Comment (PaulRBerg) [reply src/interfaces/ISablierV2Lockup.sol:138]: ok

**Pull request merged**